setup.cfg

```diff
@@ -78,6 +80,7 @@ tests =
     pytest-asyncio
     pytest-cov
     pre-commit
+    requests_mock
```

Comment:
Do we use this?

Reply:
I was using it for one test previously but removed it now.

Reply:
[Done](https://github.com/astronomer/astronomer-providers/pull/178/commits/dd0e58f6db64fbf221179b6badc10f1cc1e6f07b)